client/src/main/java/com/paypal/selion/platform/utilities/WebDriverWaitUtils.java

```diff
@@ -67,7 +64,6 @@ private static void waitForCondition(ExpectedCondition<?> condition, long timeou
      */
     public static void waitUntilElementIsClickable(final String elementLocator) {
         logger.entering(elementLocator);
-        Preconditions.checkState(Grid.driver() != null, INVALID_STATE_ERR_MSG);
```

Comment:
The contract for Grid.driver() states it will not return `null`. Thus the check is not needed. If you want to check that the `TestSession` is valid, that a different meaning and is accessible via `Grid.getTestSession()`